docs/elements/examples/shadcn-ui.mdx

```diff
@@ -506,3 +549,48 @@ export default function SignInPage() {
 }
 ```
 </CodeBlockDemo>
+
+
+## OTP Input
+
+The following example demonstrates how to make a one-time password (OTP) input with Clerk Elements.
```

Comment:
```suggestion
The following example demonstrates how to make a one-time password (OTP) input with Clerk Elements. This example will only work if placed within a `Step`, as shown in [the sign-in example](#sign-in).

```

Reply:
We also render this within the sign-up step as well. Would that change you approach to the callout here?

Reply:
Good point! I updated the test to make it clearer. What do you think?